### ISSUE TITLE
1.1 release

### ISSUE THREAD

**Issue by adamralph** [xbehave/xbehave.net#104]
**Ready** when all other issues forming part of the release are **Done**.
- [x] prepare wiki
- [x] prepare blog post
- [x] prepare website

---
- [x] ensure patches have been merged from master to dev
- [x] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [x] check build, create PR from upstream dev to master
- [x] check build, merge PR
- [x] check build, create release in GitHub UI including releaseNotes, mentioning non-owner contributors, if any
- [x] push nuget package
- [x] copy release notes from GitHub to NuGet
- [x] de-list pre-release nuget packages if present
- [x] update website with contributors list (if in place)
- [x] publish blog post
- [x] update wiki
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in JabbR (xbehavenet and general-chat)
- [x] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [x] increment minor version on branch from dev
- [x] push to origin branch, create PR to upstream dev
- [x] check build, merge PR
- [x] update samples/raise issue to do so
- [x] create a new milestone for the next release
- [x] create new issue based on this one for next release, adding it to the new milestone
- [x] close all issues on this milestone
- [x] close this milestone

**Comment by adamralph** [comment]
https://twitter.com/adamralph/status/445575888475025409
